Select ID type/script.js

```diff
@@ -0,0 +1,7 @@
+
+const button = document.getElementById('btn-phrase');
+const input = document.getElementById('input-phrase');
```

Comment:
Looks like this script is looking for an element with id of `input-phrase` but there is no html file in this folder. Please provide